cincinnati/src/plugins/mod.rs

```diff
@@ -358,7 +361,8 @@ where
     let mut io = initial_io;
 
     for next_plugin in plugins {
-        io = next_plugin.run(io).await?;
+        let span = get_tracer().start("plugin", None);
+        io = next_plugin.run(io).instrument(_span).await?;
```

Comment:
Sorry, I had assumed you'd replace the other occurrences :smile: 

```suggestion
        io = next_plugin.run(io).instrument(span).await?;
```